engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js

```diff
@@ -111,6 +111,22 @@ angular.module('Bastion.products').controller('ProductsController',
             });
         };
 
+        $scope.openAdvancedSyncModal = function () {
+            nutupane.invalidate();
+            $uibModal.open({
+                templateUrl: 'products/bulk/views/products-bulk-advanced-sync-modal.html',
+                controller: 'ProductsBulkAdvancedSyncModalController',
+                size: 'lg',
+                resolve: {
+                    bulkParams: function () {
+                        return getBulkParams();
+                    }
+                }
+            }).closed.then(function () {
+                nutupane.refresh();
```

Comment:
Do we need to refresh on close since syncs generate tasks?  In other words, nothing in the table changes after initiating the sync but only upon completion.